examples/src/main/scala/sttp/tapir/examples/status_code/customStatusCodePekko.scala

```diff
@@ -0,0 +1,53 @@
+// {cat=Status code; effects=Future; server=Pekko HTTP}: Responding with a custom status code
+
+//> using dep com.softwaremill.sttp.tapir::tapir-core:1.11.13
+//> using dep com.softwaremill.sttp.tapir::tapir-pekko-http-server:1.11.13
```

Comment:
This is obviously not correct, but I don't know which version to use in an example for a feature that has not been released yet.

Reply:
Unfortunately examples of new features have to be commented out until there's a release